src/js/controller/model.js

```diff
@@ -325,7 +325,7 @@ define([
                 _provider.volume(volume);
             }
             var mute = (volume === 0);
-            if (mute !== this.get('mute')) {
+            if (mute !== (this.get('autostartMuted') || this.get('mute'))) {
```

Comment:
Let's add a `getMute` method to model since now we've got `this.get('autostartMuted') || this.get('mute')` in at least 3 places.